Adding April Update to Docs (#43)
Ammalgam-Protocol/docs: docs/updates/April_2024.md

---
sidebar_position: 6
---

# April 2024

## tl;dr

- Interviewing a number of folks for the front end devs to help maintain
  momentum after pushing most of the current team to the contract side.
- Contract work is the main priority and the resources and attention have been
  focused to execute.

> - Planning to announce the raise once we are in Audits.

Reviewer: Audits -> audits

- Built content calendar for social to help build organic socials growth.
- Defined team Vision, Mission, and Values.

## Development

### Hiring

We have been actively interviewing candidates, thank you to anyone who connected
us with candidates or shared recruiting resources. I was able to delegate much
of the hiring process over to the team, only directly participating in final
interviews. While we max out our engineering resources to complete the contract
code, new hires will onboard to the front end and allow us to maintain momentum

> in the short term and execrate our velocity once contract are in audits. 

Reviewer: "execrate" -> accelerate?
"contract" -> "the contract" xor "the contracts"

### Contracts

Last month I mentioned that we hired a new contract engineer and are giving two
front end engineers contract work. A lot of work went into leveling up our newer
engineers to understand and contribute to our contracts and those efforts are
starting to manifest in improvements to our velocity.

To give a bit more context, we have three remaining items to wrap up:

- Interest accrual
- Cascading liquidation protections
- Liquidations

> At the start of last month, one of these items was owned by an engineer, but now

Reviewer: "one of these items was owned by an engineer" ?
maybe "one engineer"?

Author: reworded

we have the first two being handled by our strongest engineers, and I am going
to start devoting my time to the last one. Our forth contract engineer had been
working on the front end, but is now working on cleaning up some issues that

> came to light when we did a white paper audit we did with ChainSecurity in

Reviewer 2: came to light when we did a white paper audit we did with ChainSecurity in
```suggestion
came to light when we did a white paper audit with ChainSecurity in
```

February.

> While we were raising, I did a poor job supporting the engineering team. I have

Reviewer: section too harsh

Author: I was not able to do as good of a job supporting the engineering team

> the master plan in my head and often times the hardest work I have to do it

Reviewer 2: ```suggestion
the master plan in my head and often times the hardest work I have to do is
```

Author: reworded paragraph per other comments.

to use this wisely. Once we are in audits we will have eliminated most of the
development execution risks. Announcing before those risks are mediated means
that we could generate attention and then we may struggle to maintain it. Thank
you for everyone who has reached out to offer exposure in anyway, we will take
you up on all of it once we are in audits. In the meantime, I want to focus on
getting through the last bits of code we need to wrap this up.

## Business & GTM

While we are grinding on the development, we are also working to build some
organic momentum on socials.

- We developed a content calendar focused on highlighting Ammalgam’s competitive
  differentiators (capital efficiency, long / short on-chain, advanced market
  making strategies, no oracle for increased autonomy, etc.) and are automating
  Ammalgam’s Twitter account to ensure we are consistently increasing social
  media activity.
- We have also developed content and will be launching a “Building in Public”
  campaign that orchestrates the Ammalgam team’s social accounts, AMAs and
  Twitter Spaces.  Please let us know if you are interested in participating in
  AMAs or Spaces with Ammalgam.

> - We have defined Ammalgam’s mission, vision, and values.

Reviewer: capital? M, V, V

---
sidebar_position: 6
---

# April 2024

## tl;dr

> - Interviewing a number of folks for the front end devs to help maintain

Reviewer 3: Probably nit-picking here, but can you please rename `front end` to `Front-end` in another 3 instances as well?
```suggestion
- Interviewing a number of folks for the Front-end devs to help maintain
```

  momentum after pushing most of the current team to the contract side.
- Contract work is the main priority and the resources and attention have been
  focused to execute.
- Planning to announce the raise once we are in audits.
- Built content calendar for social to help build organic socials growth.

> - Defined team Vision, Mission, and Values.

Reviewer 3: Ammalgam instead of team sounds better in my head at least.
```suggestion
- Defined Ammalgam's Vision, Mission, and Values.
```

## Development

### Hiring

We have been actively interviewing candidates, thank you to anyone who connected
us with candidates or shared recruiting resources. I was able to delegate much
of the hiring process over to the team, only directly participating in final
interviews. While we max out our engineering resources to complete the contract
code, new hires will onboard to the front end and allow us to maintain momentum
in the short term and accelerate our velocity once the contracts are in audits. 

### Contracts

> Last month I mentioned that we hired a new contract engineer and are giving two

Reviewer 3: ```suggestion
Last month I mentioned that we hired a new Solidity engineer and are giving two
```

> front end engineers contract work. A lot of work went into leveling up our newer

Reviewer 3: ```suggestion
Front-end engineers contracts work. A lot of work went into levelling up our newer
```

- Liquidations

At the start of last month, interest accrual was the only item getting
attention. Now we have a plan to tackle all three with the larger team. Our
forth contract engineer had been working on the front end, but is now working on
cleaning up some issues that came to light when we did a white paper audit with
ChainSecurity in February.

While we were raising, I was not able to do as good of a job supporting the
engineering team. I have the master plan in my head and often times the hardest
work I have to do it break it down for everyone else in a comprehensible way. As
I spent more time focusing on raising funds, more pressure fell on team members
to figure things out on their own and this wasn't always as optimal as when I

> was able to be more available.

Reviewer: much better
